### PR TITLE
Chore: Add Open Doodles link to illustrations

### LINKED ISSUE #588: [Add] Open Doodles to illustrations
### Site URL

https://opendoodles.com/

### Description of the above provided site

Open Doodles is a vibrant collection of freely-usable illustrations that add personality to any project. With diverse hand-drawn characters, objects, and backgrounds, Open Doodles sparks creativity for designers, developers, and content creators. Enhance websites, presentations, and social media with these charming doodles. Let your imagination flow with the expressive world of Open Doodles.

### Category

Frontend

### Sub Category

illustrations

### Checklist

- [X] I have checked the existing issues
- [X] I have read the [Contributing Guidelines](https://github.com/rupali-codes/LinksHub/blob/main/CONTRIBUTING.md)
- [X] I am willing to work on this issue (optional)
- [ ] I am a GSSoC'23 contributor

### PR DESCRIPTION
**Pull request by Oyebolade**
## Fixes Issue
Closes #588

## Changes proposed
I added Open Doodles link to frontend category, illustrations sub-category.

## Screenshots
open doodles

### CODE REVIEW

File: database/frontend/illustrations.json [hunk 1]
```diff
@@ -1,50 +1,86 @@
...
-        "name": "Freepik",
-        "description": "Freepik goal is to offer the best high-quality content: illustrations, photos, icons, mockups, and presentations templates etc.",
-        "url": "https://www.freepik.com/",
-        "category": "frontend",
-        "subcategory": "illustrations"
-    }
+[
+  {
+    "name": "Glaze",
+    "description": "All in one resource to get free illustrations images for characters, backgrounds, avatar and more.",
```
Comment (CBID2) [inline database/frontend/illustrations.json:4]: [Typo] 
```suggestion
    "description": "All in one resource to get free illustrations images for characters, backgrounds, avatars, and more.",
```

File: database/frontend/illustrations.json [hunk 2]
```diff
@@ -1,50 +1,86 @@
...
+    "url": "https://glazestock.com/",
+    "category": "frontend",
+    "subcategory": "illustrations"
+  },
+  {
+    "name": "unDraw",
+    "description": "unDraw os another exccelent platform to get free illustration images with No color restrictions.",
```
Comment (CBID2) [inline database/frontend/illustrations.json:11]: [Typo]
```suggestion
    "description": "unDraw is another excellent platform to get free illustration images with no color restrictions.",
```

File: database/frontend/illustrations.json [hunk 3]
```diff
@@ -1,50 +1,86 @@
...
+    "url": "https://undraw.co/illustrations",
+    "category": "frontend",
+    "subcategory": "illustrations"
+  },
+  {
+    "name": "DrawKit",
+    "description": "It is a collection of fully customisable PSD illustrations, in two styles, available to use on your next project.",
```
Comment (CBID2) [inline database/frontend/illustrations.json:18]: [Proofreading] What are the two styles that you mentioned? I think adding this information could help developers understand why they should use this source. :)  
```suggestion
    "description": "This is a collection of fully customizable PSD illustrations. They are done in two styles, and are available to use in your next project.",
```

File: database/frontend/illustrations.json [hunk 4]
```diff
@@ -1,50 +1,86 @@
...
+    "url": "https://drawkit.com/",
+    "category": "frontend",
+    "subcategory": "illustrations"
+  },
+  {
+    "name": "ManyPixels",
+    "description": "Many pixel illustration 2500+ royalty free illustrations to power up your designs.",
```
Comment (CBID2) [inline database/frontend/illustrations.json:25]: [Typo] 
```suggestion
    "description": "This site contains 2500+ pixel and  royalty-free illustrations you can use to power up your designs.",
```

File: database/frontend/illustrations.json [hunk 5]
```diff
@@ -1,50 +1,86 @@
...
+    "url": "https://www.manypixels.co/gallery/",
+    "category": "frontend",
+    "subcategory": "illustrations"
+  },
+  {
+    "name": "Reshot",
+    "description": "Free Icons & Illustrations design freely with instant downloads and commercial licenses.",
```
Comment (CBID2) [inline database/frontend/illustrations.json:32]: [Typo]
```suggestion
    "description": "Free Icons & Illustrations designs with instant downloads and commercial licenses.",
```

File: database/frontend/illustrations.json [hunk 6]
```diff
@@ -1,50 +1,86 @@
...
+  {
+    "name": "StorySet",
+    "description": "Awesome free customizable illustrations for your next project.",
+    "url": "https://storyset.com/",
+    "category": "frontend",
+    "subcategory": "illustrations"
+  },
+  {
+    "name": "Freepik",
+    "description": "Freepik goal is to offer the best high-quality content: illustrations, photos, icons, mockups, and presentations templates etc.",
```
Comment (CBID2) [inline database/frontend/illustrations.json:46]: [Typo]
```suggestion
    "description": "Freepik's goal is to offer the best high-quality content: illustrations, photos, icons, mockups, presentations templates, etc.",
```

File: database/frontend/illustrations.json [hunk 7]
```diff
@@ -1,50 +1,86 @@
...
+    "url": "https://www.freepik.com/",
+    "category": "frontend",
+    "subcategory": "illustrations"
+  },
+  {
+    "name": "Blush Design",
+    "description": "Blush design is a creative platform offering a diverse collection of customizable illustrations for designers, developers, and content creators. With an extensive library of vibrant artwork, users can effortlessly enhance their projects. The platform allows easy customization of colors, styles, and elements to suit individual needs, empowering creativity and enhancing visual communication.",
```
Comment (CBID2) [inline database/frontend/illustrations.json:53]: [Typo]
```suggestion
    "description": "Blush Design is a creative platform offering a diverse collection of customizable illustrations for designers, developers, and content creators. With an extensive library of vibrant artwork, users can effortlessly enhance their projects. The platform allows easy customization of colors, styles, and elements to suit individual needs, empowering creativity and enhancing visual communication.",
```

File: database/frontend/illustrations.json [hunk 8]
```diff
@@ -1,50 +1,86 @@
...
+    "url": "https://blush.design/",
+    "category": "frontend",
+    "subcategory": "illustrations"
+  },
+  {
+    "name": "UI Store Design",
+    "description": "UI store design is a platform dedicated to providing a wide range of design resources for UI/UX professionals. It offers a curated collection of high-quality templates, icons, illustrations, and other assets to enhance the visual appeal and user experience of digital products. With a focus on simplicity and efficiency.",
```
Comment (CBID2) [inline database/frontend/illustrations.json:60]: [Typo]
```suggestion
    "description": "UI Store Design is a platform dedicated to providing a wide range of design resources for UI/UX professionals. It offers a curated collection of high-quality templates, icons, illustrations, and other assets to enhance the visual appeal and user experience of digital products with a focus on simplicity and efficiency.",
```

File: database/frontend/illustrations.json [hunk 9]
```diff
@@ -1,50 +1,86 @@
...
+  {
+    "name": "Skribbl",
+    "description": "Enrich your creative projects and support free access to art of education with the collection of hand-drawn illustrations.",
+    "url": "https://weareskribbl.com/",
+    "category": "frontend",
+    "subcategory": "illustrations"
+  },
+  {
+    "name": "Humaaans",
+    "description": "Humaaans is a flexible illustration library that simplifies visual storytelling. With diverse customizable human figures, it offers a wide range of poses and clothing options for inclusive and unique illustrations. Create captivating visuals effortlessly for websites, presentations, and more. Customize, mix, and match vector illustrations to bring your stories to life.",
```
Comment (CBID2) [inline database/frontend/illustrations.json:74]: [praise] Great job! :) This describes the benefits of using this source in a detailed way! 👍🏾 
```suggestion
    "description": "Humaaans is a flexible illustration library that simplifies visual storytelling. With diverse customizable human figures, it offers a wide range of poses and clothing options for inclusive and unique illustrations. Create captivating visuals effortlessly for websites, presentations, and more. Customize, mix, and match vector illustrations to bring your stories to life.",
```

File: database/frontend/illustrations.json [hunk 10]
```diff
@@ -1,50 +1,86 @@
...
+  {
+    "name": "Reshot",
+    "description": "Free Icons & Illustrations design freely with instant downloads and commercial licenses.",
+    "url": "https://www.reshot.com/",
+    "category": "frontend",
+    "subcategory": "illustrations"
+  },
+  {
+    "name": "StorySet",
+    "description": "Awesome free customizable illustrations for your next project.",
```
Comment (CBID2) [inline database/frontend/illustrations.json:39]: [Praise] This describes the source in a short manner. Mentioning that it's customizable would intrigue developers who are looking for something easy to work with, especially if they are doing projects with short deadlines! :) Well done! :) 
```suggestion
    "description": "Awesome free customizable illustrations for your next project.",
```

File: database/frontend/illustrations.json [hunk 11]
```diff
@@ -1,50 +1,86 @@
...
+  {
+    "name": "Humaaans",
+    "description": "Humaaans is a flexible illustration library that simplifies visual storytelling. With diverse customizable human figures, it offers a wide range of poses and clothing options for inclusive and unique illustrations. Create captivating visuals effortlessly for websites, presentations, and more. Customize, mix, and match vector illustrations to bring your stories to life.",
+    "url": "https://humaaans.com/",
+    "category": "frontend",
+    "subcategory": "illustrations"
+  },
+  {
+    "name": "Open Doodles",
+    "description": "Open Doodles is a vibrant collection of freely-usable illustrations that add personality to any project. With diverse hand-drawn characters, objects, and backgrounds, Open Doodles sparks creativity for designers, developers, and content creators. Enhance websites, presentations, and social media with these charming doodles. Let your imagination flow with the expressive world of Open Doodles.",
```
Comment (CBID2) [inline database/frontend/illustrations.json:81]: [praise] I like how you mention that these illustrations enhance projects! :) Well done! 👍🏾 
```suggestion
    "description": "Open Doodles is a vibrant collection of freely-usable illustrations that add personality to any project. With diverse hand-drawn characters, objects, and backgrounds, Open Doodles sparks creativity for designers, developers, and content creators. Enhance websites, presentations, and social media with these charming doodles. Let your imagination flow with the expressive world of Open Doodles.",
```

**Review by CBID2 — CHANGES_REQUESTED** [review]
Hi @Oyebolade! 😄 Aside from the proofreading and typos that I mention in my inline comments(In the future, consider running your descriptions [Grammarly](https://www.grammarly.com/how-grammarly-works) before adding them to your PRs), your PR is off to a great start(note: you can the inline comments that are labeled [praise])! :)

**Comment by Oyebolade** [comment]
> Hi @Oyebolade! 😄 Aside from the proofreading and typos that I mention in my inline comments and your branch's conflicts(In the future, consider running your descriptions [Grammarly](https://www.grammarly.com/how-grammarly-works) before adding them to your PRs), your PR is off to a great start(note: you can the inline comments that are labeled [praise])! :)

Thanks @CBID2
I really appreciate the observations, corrections and suggestions. 
I didn't get the last line of your message.

**Comment by rupali-codes** [comment]
@Oyebolade please make the suggested changed and resolve conflicts :)

**Comment by Oyebolade** [comment]
> @Oyebolade please make the suggested changed and resolve conflicts :)

All right

**Pull request closed without merging**